routers/web/repo/repo.go

```diff
@@ -599,6 +599,13 @@ func SearchRepo(ctx *context.Context) {
 
 	results := make([]*api.Repository, len(repos))
 	for i, repo := range repos {
+		if err = repo.GetOwner(ctx); err != nil {
```

Comment:
Why is this code required here? `SearchRepository` should load this. Otherwise we should make this one database query.

Reply:
Good catch, I will revert the lines.